equinix/fabric_port_read_schema.go

```diff
@@ -307,13 +307,10 @@ func readFabricPortResourceSchema() map[string]*schema.Schema {
 				Schema: readFabricPortEncapsulation(),
 			},
 		},
-		"lag": {
-			Type:        schema.TypeSet,
+		"lag_enabled": {
```

Comment:
This is also a breaking change and we should deprecate the old field before removing it: https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#renaming-a-computed-attribute

Reply:
@thogarty Added a fix for all the deprecated fields you pointed out. However, the tests are failing now and we are looking into it.

Reply:
This is now fixed with the latest commit. Please check and approve.